libraries/ui-library/src/components/six-radio/six-radio.tsx

```diff
@@ -23,19 +23,19 @@ let id = 0;
   shadow: true,
 })
 export class SixRadio {
-  inputId = `radio-${++id}`;
-  labelId = `radio-label-${id}`;
-  input: HTMLInputElement;
+  private inputId = `radio-${++id}`;
+  private labelId = `radio-label-${id}`;
+  private nativeInput?: HTMLInputElement;
 
-  @Element() host: HTMLSixRadioElement;
+  @Element() host!: HTMLSixRadioElement;
 
   @State() hasFocus = false;
 
   /** The radio's name attribute. */
-  @Prop() name: string;
+  @Prop() name = '';
 
   /** The radio's value attribute. */
-  @Prop() value: string;
+  @Prop() value = 'on';
```

Comment:
Was this intentionally set to `on`? 🤔

Reply:
I set this to `on`, because this is the the default value of the native radio element. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio#data_representation_of_a_radio_group.